src/components/dialogs/notificationDialogUtils/NotificationCollaborationEditRevokeDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { get } from 'lodash-es';
+import { notificationSchema } from '../../../constants/notificationSchema';
+import NotificationDetailsDialog from '../NotificationDetailsDialog';
+
+export default function NotificationCollaborationEditRevokeDialog({
+  notification,
+  onClose,
+  open,
+}) {
+  const notificationType = notification?.message_type;
+  const currentNotificationSchema = get(
+    notificationSchema,
+    notificationType,
+  );
+  const avaialableButtons = [];
+  return (
+    <NotificationDetailsDialog
+      open={open}
+      onClose={onClose}
+      notification={notification}
+      titleId={get(currentNotificationSchema, 'titleId')}
+      moreDetailedDescription={get(
+        currentNotificationSchema,
+        'moreDetailedDescription',
+      )}
+      buttons={avaialableButtons}
+      isError={false}
+      error=""
```

Comment:
error = ""?

Reply:
So, for several of these dialogs, no buttons and therefore no api calls are required. I suppose I could pass null? Or have them default to null in notificationDialog? Same for the similar comments below.

Reply:
Exactly, just use default values

Reply:
Done